### PR TITLE
Integrated Material design components UI into Stopwatch Sample.

### PR DESCRIPTION
**Pull request by santhoshvaddi**

### CODE REVIEW

**Comment by timburks** [comment]
I pulled and ran the sample in your PR - the MDC components look good and the sample successfully called the API. Great progress!

A couple of notes:

- To merge your work into our official samples, your PRs should be against the https://github.com/googleapis/ios-docs-samples repo and not your personal fork. 

- Right now the updated sample doesn't provide any indication when the app is listening. The Assistant app on iOS uses a nice animation for this - I don't think we should put that in the dialogflow sample, but something should show that the app is listening and perhaps allow cancellation. A good way to do this might be to replace the bottom button (when the app is listening) with a new button that says "Listening.... (press to cancel)".

- The Assistant app shows partial speech results as they stream back. I think you could also do this in the sample and that it would be valuable.

- When the dialog gets to the bottom of the screen, the scrollview doesn't automatically scroll to display new output. This leads to confusion about whether or not the app is working.

**Comment by nnegrey** [comment]
* On the first one, I made the call to merge here as I wasn't sure if we would be putting this in ios-docs-samples yet. 

* For the partial speech results, I believe the Assistant App is probably using a local ML model to do the Speech-To-Text. For the Dialogflow audio request and sample. I thought we might rely on the Dialogflow's response as that can keep the sample simple so that people can easily parse out the Dialogflow components of the code. Otherwise, we'd have to add a speech-to-text component between the Dialogflow request.

**Comment by santhoshvaddi** [comment]
@timburks Thanks for the feedback. on the fourth one, True i will make the necessary changes to the code, When the dialog gets to the bottom of the screen the scrollview should automatically scroll up and display the new output. @nnegrey for the second one, Yes i agree and need some inputs from you to before i start the work.

**Comment by timburks** [comment]
For partial speech results, the Dialogflow StreamingDetectIntent method streams a reply that includes partially-recognized text. See the [StreamingDetectIntentResponse](https://cloud.google.com/dialogflow-enterprise/docs/reference/rpc/google.cloud.dialogflow.v2#google.cloud.dialogflow.v2.StreamingDetectIntentResponse) message description.

**Comment by nnegrey** [comment]
Right, sorry, I was referring to the audio request. For streaming, we should use the partial results. 

1. Demonstrate Text
2. Demonstrate Audio
3. Demonstrate Streaming Audio

**Comment by santhoshvaddi** [comment]
> For partial speech results, the Dialogflow StreamingDetectIntent method streams a reply that includes partially-recognized text. See the [StreamingDetectIntentResponse](https://cloud.google.com/dialogflow-enterprise/docs/reference/rpc/google.cloud.dialogflow.v2#google.cloud.dialogflow.v2.StreamingDetectIntentResponse) message description.

Sure @timburks I will work on it.

File: dialogflow/stopwatch/Stopwatch/DialogFlowViewController.swift [hunk 1]
```diff
@@ -0,0 +1,299 @@
...
+    func stopListening() {
+        _ = AudioController.sharedInstance.stop()
+        StopwatchService.sharedInstance.stopStreaming()
+        //        button.setTitle("Start Listening", for: .normal)
+        listening = false
+    }
+    func processSampleData(_ data: Data) -> Void {
+        audioData.append(data)
+        
+        // We recommend sending samples in 100ms chunks
```
Comment (pskirko) [inline dialogflow/stopwatch/Stopwatch/DialogFlowViewController.swift]: Can you add a link back to the public documentation where this is specified? Thanks.

File: dialogflow/stopwatch/Stopwatch/DialogFlowViewController.swift [hunk 2]
```diff
@@ -0,0 +1,299 @@
+//
+//  DialogFlowViewController.swift
```
Comment (pskirko) [inline dialogflow/stopwatch/Stopwatch/DialogFlowViewController.swift]: Note, all the code should be Dialogflow, not DialogFlow. My understanding is that TensorFlow is camel-cased, and Dialogflow is not, on purpose, and that distinction should be maintained.

Comment (santhoshvaddi) [reply dialogflow/stopwatch/Stopwatch/DialogFlowViewController.swift]: Thanks @pskirko I will rename it to "Dialogflow"

Comment (timburks) [reply dialogflow/stopwatch/Stopwatch/DialogFlowViewController.swift]: @santhoshvaddi thanks for fixing these!

**Comment by timburks** [comment]
LGTM - please merge this to master and make a PR to the main repo.

**Pull request merged**